### PR TITLE
Use single options parameter as argument for functions - Close #337

### LINKED ISSUE #337: Standardise the arguments’ order along functions
Ideally we should follow the same order for the arguments along all the existing functions. I propose to have `secret, second secret, asset data, timeOffset`.

### PR DESCRIPTION
**Pull request by alepop**
Use single option parameter as an argument for functions in these `transaction` modules:
- [x] `send`
- [x] `sendFromMultisignatureAccount`
- [x] `registerSecondSignature`
- [x] `registerDelegate`
- [x] `castVotes`
- [x] `registerMultisignatureAccount`
- [x] `createDapp`
- [x] `ransferIntoDapp`
- [x] `transferOutOfDapp`

close #337

### CODE REVIEW

File: test/transactions/1_registerSecondSignature.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ describe('#registerSecondSignature transaction', () => {
 			});
 
 			it('should have the correct publicKey if the provided second secret is an empty string', () => {
-				registerSecondSignatureTransaction = registerSecondSignature('secret', '');
+				registerSecondSignatureTransaction = registerSecondSignature({ secret: 'secret', secondSecret: '' });
```
Comment (Tosch110) [inline test/transactions/1_registerSecondSignature.js]: Please use a const here for secret as in the other test files

File: src/transactions/0_sendFromMultisignatureAccount.js [hunk 2]
```diff
@@ -23,20 +23,21 @@ import slots from '../time/slots';
...
- * @param recipientId string
- * @param amount number
- * @param secret secret
- * @param secondSecret secret
- * @param requesterPublicKey string
- * @param timeOffset number
+ * @method sendFromMultisignatureAccount
+ * @param {Object} Object - Object
+ * @param {String} Object.recipientId
+ * @param {Number} Object.amount
```
[the remark this answers is not in the document]
Comment (alepop) [reply src/transactions/0_sendFromMultisignatureAccount.js]: But in tests `amount` is a `number`.

[the remark this answers is not in the document]
Comment (willclarktech) [reply src/transactions/0_sendFromMultisignatureAccount.js]: Yeah, this is a change that hasn't yet been reflected in the tests. We should make an issue.

[the remark this answers is not in the document]
Comment (alepop) [reply src/transactions/0_sendFromMultisignatureAccount.js]: I can change it in this PR.

[the remark this answers is not in the document]
Comment (willclarktech) [reply src/transactions/0_sendFromMultisignatureAccount.js]: https://github.com/LiskHQ/lisk-js/issues/409

**Pull request merged**